README.md

```diff
@@ -287,11 +287,30 @@ Normal build is
 
     cmake .. -DCMAKE_TOOLCHAIN_FILE=~/mjbots/kodlab_mjbots_sdk/cmake/pi.cmake
 
+
+
 # Running Code
 To Run code, first scp the binary onto the pi, and then Run it as sudo using:
 
     sudo ./code_binary
 
+# Troubleshooting 
+If you encounter the below a pthread error during Cmake keep the below commands in the main repo directory CMakeLists.txt after the endif()
+
+    set(CMAKE_THREAD_LIBS_INIT "-lpthread")
```

Comment:
Why are these changes not made in the `CMakeList.txt`? Does it break it for other people?

Reply:
These changes don't break it, but @ethanmusser said that as we are looking to support the sdk for 20.04 and these occurred on 18.04 they are not required in the CMakeList.txt.

Reply:
If it doesn't break 20.04, then you should add it to the `CmakeList.txt`. It'll be a pain for you to develop with that unmerged change.